libs/test/SalesforceSDKTest/src/com/salesforce/androidsdk/auth/OAuth2Test.java

```diff
@@ -317,7 +338,52 @@ public void testGetAuthorizationUrlWithScopes() throws URISyntaxException {
 
         //empty scopes -- should not find scopes
         tryScopes(new String[] {}, null);
+
+        //null scopes -- should not find scopes
+        tryScopes(null, null);
```

Comment:
Making sure null scopes are handled - null scopes can now happen with the change to BootConfig.java